Art/anorme-loading/style.css

```diff
@@ -0,0 +1,93 @@
+:root {
+	--color1: pink;
+	--color2: beige;
+	--color3: lightcyan;
+}
+p {
+	margin: 0px;
+}
+
+h1 {
+	text-align: center;
+	height: 20px;
+	margin: 20px auto;
+	color:lightsteelblue;
+}
+
+.wrapper {
+	display: flex;
+	margin:40px auto ;
+	background-color: lightslategray;
+	position: relative;
+/*	top: 200px;*/
```

Comment:
# Commented style
Please remove the commented lines

Reply:
I've removed the commented code as well. Do I need to submit another separate PR? or should I just commit and push upstream?